students/jiwon/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    name = models.CharField(max_length=10)
+    email = models.CharField(max_length=30)
```

Comment:
- 전체적으로 max_length가 작은 것 같습니다. 아래와 같이 조금씩 늘려주세요.
- =을 기준으로 정렬해 주세요.(wecode convention 입니다.)

```suggestion
    name  = models.CharField(max_length=50)
    email = models.CharField(max_length=100)
```